coremltools/converters/nnssa/coreml/graph_pass/mlmodel_passes.py

```diff
@@ -244,27 +245,49 @@ def remove_redundant_transposes(spec):
     that compose to the identity.
     """
 
+    def _get_input_output_hashmap(nn_layers):
+        # Hashmap for layers
```

Comment:
Add comment on the return types of this function:

`output_to_layers`: [str: layer proto message] : blob name: layers that it feeds into. 

Based on this description, don't you think, `blob_name_to_layers` is a better term?

Reply:
good!